todo_app_tests/integrationtests/test_app.py

```diff
@@ -0,0 +1,49 @@
+import os
+import pytest
+import requests
+from dotenv import load_dotenv, find_dotenv
+from todo_app import app
+
+@pytest.fixture
+def client():
+    # Use our test integration config instead of the 'real' version
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+    # Create the new app.
+    test_app = app.create_app()
+    # Use the app to create a test_client that can be used in our tests.
+    with test_app.test_client() as client:
+        yield client
+
+def test_index_page(monkeypatch, client):
+    monkeypatch.setattr(requests, 'get', get_lists_stub)
+    response = client.get('/')
+    assert response.status_code == 200
```

Comment:
Would be nice to test at least that the test data has come through, e.g.
```python
assert 'Test card' in response.data.decode()
```

Reply:
Added this line